index.d.ts

```diff
@@ -0,0 +1,32 @@
+type Mode = "binary" | "metric" | "decimal" | "compatibility";
+
+interface ParseOptions {
+    mode?: Mode;
+}
+
+declare function parse(value: number | string, options?: ParseOptions): number | null;
+
+interface FormatOptions {
+    decimalPlaces?: number;
+    fixedDecimals?: number;
+    mode?: Mode;
+    thousandsSeparator?: string;
+    unit?: string;
```

Comment:
Could this not be a locked down list of types? similar to how "Mode" is defined?

Reply:
Oh yeah, I'll use the units from the readme as a reference

Reply:
Speaking of the readme, do you mind if I lob some changes to the readme into this PR? I found the compatibility mode explanation for example confusing since it talks about overwriting the decimal units with the commonly misused (binary) "prefixes", but then lists the correct units again. The description of the `unit` param also omits the units this fork is all about.